internal/pkg/cli/interfaces.go

```diff
@@ -306,8 +306,15 @@ type zipAndUploader interface {
 	ZipAndUpload(bucket, key string, files ...s3.NamedBinary) (string, error)
 }
 
+type Uploader interface {
+	zipAndUploader
+	Upload(bucket, key string, file s3.NamedBinary) (string, error)
+}
+
 type customResourcesUploader interface {
 	UploadEnvironmentCustomResources(upload s3.CompressAndUploadFunc) (map[string]string, error)
+	UploadRequestDrivenWebServiceCustomResources(upload s3.CompressAndUploadFunc) (map[string]string, error)
+	UploadRequestDrivenWebServiceLayers(upload s3.UploadFunc) (map[string]string, error)
```

Comment:
**feel free to ignore this comment**: I'm just thinking out loud 🤔 , we have a lot of interfaces in `cli` that are large. Ideally most interfaces that we define should have one or two fairly generic methods. Moving forward we should consider large interfaces as a potential sign for a codesmell.

So how can we make the template pkg better? Maybe we can move these to functions 💭 
```go
func ParseParseLoadBalancedWebService(tpl ReadParser, dat *LoadBalancedWebServiceOpts)
```